### PR TITLE
Update doc with APIM 4.2.0 and IS 6.0.0 compatibility

### PR DESCRIPTION
**Pull request by DinithiDiaz**
## Purpose
This PR is to update the UK Toolkit documentation with APIM 4.2.0 and IS 6.0.0 compatibility.
Reference: https://github.com/wso2/docs-open-banking/issues/772

### CODE REVIEW

File: en/docs/get-started/dynamic-client-registration.md [hunk 1]
```diff
@@ -14,6 +14,25 @@ This page explains how to onboard TPP applications using the Dynamic Client Regi
 
     3. Restart the Identity Server.
 
+    4. If you are using **WSO2 API Manager 4.2.0**, change the API Manager REST API version.
```
Comment (ChinthakaJ98) [inline en/docs/get-started/dynamic-client-registration.md:17]: What about the previous versions? Dont we need to add what versions need to be used?

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 2]
```diff
@@ -263,7 +263,20 @@ account retrieval. By default, this is disabled and the configuration is set to
     password="$ref{super_admin.password}"	
     server_url = "{tcp://<SI_HOST>:7612}"	
     ```  
-   
+
+16. If you are using WSO2 API Manager 4.2.0, you need to change the API Manager REST API version from V2 to V3.
```
Comment (ChinthakaJ98) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md:267]: AFAIK in the default configs the version is not mentioned. So we will not be able to say to change it. But we can mention to use a the relevant endpoint version based on the APIM version. @Ashi1993 WDYT?

**Review by ChinthakaJ98 — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 2 above)
Comment (Ashi1993) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md:267]: Default value is read from the J2 file hence we dont have to menion the default values.

**Review by Ashi1993 — APPROVED** [review]

File: en/docs/get-started/dynamic-client-registration.md (hunk 1 above)
Comment (ChinthakaJ98) [reply en/docs/get-started/dynamic-client-registration.md:17]: I guess we will have to include something similar to the change made in configuring-api-manager-for-ob.md page

**Review by ChinthakaJ98 — APPROVED** [review]

**Pull request merged**